packages/amplify-cli/src/plugin-helpers/scan-plugin-platform.ts

```diff
@@ -58,13 +60,21 @@ export async function scanPluginPlatform(pluginPlatform?: PluginPlatform): Promi
   pluginPlatform!.lastScanTime = new Date();
   await writePluginsJsonFile(pluginPlatform!);
 
+  await checkPlatformHealth(pluginPlatform);
```

Comment:
We don't use the result of the function for anything. What happens if we fail to find core plugins, how will we exit the process to make sure not running into further errors down the execution path because of the missing plugins?

Reply:
The additional health check here is only meant to check on it, not for fixing problems. The messages printed out will help the customer and us to identify and fix the issues.

Reply:
do we need a try catch here - to exit the process?

Reply:
health check normally does not throw errors, it only prints out abnormal things that it found in the plugin platform